app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::GreetingsController < ApplicationController
+  def index
+    render json: Greeting.all.sample
```

Comment:
- [ ] Good job so far! Request from the frontend is hitting the API endpoint at the backend successfully. However, random greetings are not displayed on the screen as intended. The browser console shows that CORS is not properly configured. Let's ensure to improve upon this aspect.

**Error by CORS:**
![image](https://user-images.githubusercontent.com/12745474/228416121-be162369-435c-474a-891c-f9f0f0437eac.png)

Reply:
It is configured correctly. Please next time take time to read the file thoroughly https://github.com/wassimchakib/hello-rails-back-end/blob/c84e7d49bc9c1a53f34857f8f668b0a68aa5b0f2/config/application.rb#L21. It is in `application.rb`, you changed the port to 3001 that's why it didn't work on your side. It doesn't mean that it's not configured

Reply:
**For the next reviewer**, to make things easier since both react and rails use port 3000, i decided to refactor the front end app to call rails API endpoints with port 1700. So in order to test the app, please run `rails server --port=1700`

Reply:
* Hey @wassimchakib  👋
* The error mentioned by the previous CR still exists, I have tried all the instructions on your readme files and also the one above ⬆️ still the random greetings doesn't show, Kindly look into it check your cors settings and update your setup instructions to a working one  
![image](https://user-images.githubusercontent.com/52497246/228429602-16e02486-cadf-456d-8d15-78e252d3aa3a.png)

Reply:
@Davidosky007 it is working on my side, i tested it 3 times and it works. 
![image](https://user-images.githubusercontent.com/50754458/228430773-c5b5babd-4afc-4c5e-8060-6b4c8ef74ea4.png)

Reply:
As I mentionned again, since React and Rails use the same port. I changed the rails port to 1700. And you can clearly see in the `application.rb` the CORS configuration if you did this project before, you will know what I'm talking about. If you can't make it work on your side, it doesn't mean it's my issue. Thanks for understanding!